autode/neb/original.py

```diff
@@ -698,7 +698,7 @@ def peak_species(self) -> Optional[Species]:
             return None
 
         image = self.images[self.images.peak_idx]
-        return image.new_species()
+        return image.copy()
```

Comment:
It's a possibility. I imagine I used `new_species` rather than `copy` for _a_ reason at the time, which alludes me now. Thanks for looking, let's leave it – I trust your scan through!